packages/mui-material/src/Autocomplete/Autocomplete.spec.tsx

```diff
@@ -171,3 +172,33 @@ function CustomListboxRef() {
     >(e);
   }}
 />;
+
+type PaperComponentProps = { title?: string };
+declare module '@mui/material/Paper' {
+  export interface PaperBaseProps extends PaperComponentProps {}
+}
+
+function PaperComponent({ title, children, ...props }: PaperProps) {
+  return (
+    <Paper {...props}>
+      {title}
+      {children}
+    </Paper>
+  );
+}
+
+function AutocompleteCustomPaper() {
+  return (
+    <Autocomplete
+      options={['one', 'two', 'three']}
+      renderInput={(params) => <TextField {...params} />}
+      PaperComponent={PaperComponent}
+      componentsProps={{
+        paper: {
+          title: 'Title',
+          variant: 'outlined',
+        },
+      }}
+    />
+  );
+}
```

Comment:
This isn't testing what's needed. Because if I revert the PaperComponent type back to `React.JSXElementConstructor<React.HTMLAttributes<HTMLElement>>`, the test still passes.

Reply:
Hello again! Sorry for the late response. It should be fixed now